sa/model.go

```diff
@@ -281,12 +281,16 @@ type regModel struct {
 	Agreement string `db:"agreement"`
 	// InitialIP is stored as sixteen binary bytes, regardless of whether it
 	// represents a v4 or v6 IP address.
+	//
+	// Deprecated: This field is no longer used and will be removed in a future.
 	InitialIP []byte    `db:"initialIp"`
 	CreatedAt time.Time `db:"createdAt"`
 	LockCol   int64
 	Status    string `db:"status"`
 }
 
+var defaultInitialIP = net.ParseIP("0.0.0.0").To16()
```

Comment:
Either:
a) inline this on line 331 -- this variable is only used in one place so there's no need for it to be a variable; or
b) change the name -- "default" implies that it can be overridden.

In either case, also add a comment explaining that we no longer need to record the IP but we still have to hand the database _something_ since the column is "NOT NULL".